### PR TITLE
Added instructions for RPM Based Systems

### PR DESCRIPTION
**Pull request by BClark09**
Note: Not to be merged until RPM Repo is fully tested, PR is for feedback on changes and review.

 - Also added collapsible includes for maintainability.

image

image

Signed-off-by: Ben Clark <ben@benjyc.uk>

### CODE REVIEW

**Comment by BClark09** [comment]
WIP removed! If people are happy with the RPM instructions (and the collapsible) then I can announce the RPM snapshot repo and point towards this page.

**Comment by ThomDietrich** [comment]
Hmm not as clean as I hoped it would be but I see that you already tried to make it as clean as possible ;) 
Just to be clear, WIP removed as in "Ready to be merged"?

**Comment by BClark09** [comment]
Ready to be merged indeed! Although if I can make it cleaner I will certainly try! What did you mean when you said clean? The produced output or the {%include... stuff?

**Comment by ThomDietrich** [comment]
Totally forgot about this PR, sorry.
No problem, it's certainly clean. It's obvious you tried to make everything as uniform as possible. Thanks!!

File: installation/linux.md [hunk 1]
```diff
@@ -184,22 +286,32 @@ The most important commands to control the openHAB service are given below.
...
+
 Upgrading is as easy as:
 
 ```shell
 sudo apt-get update
 sudo apt-get upgrade
 ```
 
-#### Changing Versions
+*Changing Versions*
```
Comment (ThomDietrich) [inline installation/linux.md]: I used to link to this headline. If we were to add an anchor here, would the collapsible open automatically?

File: installation/linux.md [hunk 2]
```diff
@@ -114,30 +153,83 @@ If you plan on disconnecting your machine from the internet, then you will want
...
+sudo /etc/init.d/openhab2 start
+sudo /etc/init.d/openhab2 status
+
+sudo update-rc.d openhab2 defaults
+```
+
+{% include collapsible/item-end.html %}
+{% include collapsible/heading.html %}
+
+Systems based on **systemd** (e.g. Debian 8, Ubuntu 15.x, Raspbian Jessie and newer):
```
Comment (ThomDietrich) [inline installation/linux.md]: Bug: Markdown bold format not working here

**Pull request merged**